source-container-build/app/requirements-dev.in

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+
+pytest
+pytest-cov
```

Comment:
Btw, why do we need pytest if the tests are implemented with the stdlib `unittest`?

Reply:
It's just a choice of test runners and is bound to the selected test framework used to implement the test. By the way, I like the way with pytest to run tests and output kinds of coverage with a single command and it's much straightforward.

Reply:
Ok, that makes sense. To be honest, I don't even know how to run tests with unittest :smile: